backend/src/types.ts

```diff
@@ -46,3 +52,33 @@ export interface IImage {
 export type ManagerAPI = {
     setData: (extensionId: string, collections: ICollection[], items: IItem[]) => void;
 }
+
+let action: IFileAction = {
+    uri: undefined,
+    actionType: ActionType.File
+}
+let item: IItem = {
+    id: "",
+    title: "",
+    description : "",
+    labels: [],
+    action1: {
+        name: "",
+        action: {
+            actionType: ActionType.Execute,
+            executeAction: () => { return Promise.resolve();}
+        },
+        contexts: [{
+            project: "",
+            params: 3
+        }]
+    },
+    action2: {
+        name: "",
+        action: action,
+        contexts: [{
+            project: "",
+            params: 3
+        }]
+    }
+}
```

Comment:
This is an example and should be removed

Reply:
Of course. It's temporary here.